webapp/src/main.ts

```diff
@@ -5,6 +5,8 @@ import {AppModule} from './app/app.module';
 import {environment} from './environments/environment';
 import {Globals} from './app/shared/globals';
 
+import 'hammerjs';
```

Comment:
why is hammerjs imported?

Reply:
Mostly because Angular Material wants to have it and throws a warning otherwise (see [Angular Material docs](https://material.angular.io/guide/getting-started#step-5-gesture-support) ). It is just a library to improve touch support and looking at the source code it is so small that it would not hurt at all to include it. I am not sure it it is actually used by the components we are using but it's a possibility.

Reply:
If the question was: Why was it specifically imported here? This ensures that the treeshaking doesn't remove the dependency in the build.

Reply:
Alright, makes sense.